### ISSUE TITLE
Hosted app is broken with 502 / connection timeout

### ISSUE THREAD

**Issue by mcgarrah** [mcgarrah/legendary_quick_quiz#10]
Error on deployed app.

**Comment by mcgarrah** [comment]
Deployment is hosed on the changes the the webroot/__init__.py file

``` text
2024-12-15T20:26:13.679000 - File "/srv/app.py", line 8, in <module>
2024-12-15T20:26:13.679000 - from . import __version__, __build_date__
2024-12-15T20:26:13.679000 - ImportError: attempted relative import with no known parent package
```

This only shows up on the qunicorn deployment environment. Local debugging does not have this issue.

**Comment by mcgarrah** [comment]
Root issue was moving my `app.py` Flask entry point to a subdirectory. Nothing I did would make the deployment work unless I moved it back to webroot.  FLASK_APP in a dotEnv and a variety of other things didn't help.

The direct error above was solved with importing from `__init__` rather than the `.` which was another issue that circled this same one with having an application name.

Issue resolved in [v0.1.12 ](https://github.com/mcgarrah/legendary_quick_quiz/releases/tag/v0.1.12)
